specifications/M1/issuance.flow.puml

```diff
@@ -5,10 +5,15 @@
 
 autonumber
 
+box "Client Organization"
 participant "Client" as C
-participant "Client \nCredential Service" as CS
-participant "Client \nSecure Token Service" as STS
+participant "Credential Service" as CS
+participant "Secure Token Service" as STS
+end box
+
+box "Issuer Organization"
```

Comment:
Just a thought: using the word "Organization" here could mislead readers to think, that the "issuer" is _just another company_, where in actuality it's likely a (or one of several) specially equipped entities.
Maybe "Issuing Entity"? "Issuer Entity"?

Reply:
I think it's still an organization, but fine for me. Same for the 'Client' or stick with 'Organization' there?
Will change it to 'Issuer Entity'.

Reply:
I think we also need to update to 'Client Entity'. Otherwise, we need to explain the difference.

Reply:
My point was to distinguish between the "Organisation", i.e. the participant, which can exist many times in a data space, and he issuer, which exists once, or only a few times. 

Just so there is a distinction between these two.

Reply:
Issuer Entity is fine for me